topostats/grains.py

```diff
@@ -742,3 +795,741 @@ def keep_largest_labelled_region(
         sizes = np.array([(labelled_image == label).sum() for label in range(1, labelled_image.max() + 1)])
         # Keep only the largest region
         return np.where(labelled_image == sizes.argmax() + 1, labelled_image, 0).astype(bool)
+
+    @staticmethod
+    def flatten_multi_class_tensor(grain_mask_tensor: npt.NDArray) -> npt.NDArray:
+        """
+        Flatten a multi-class image tensor to a single binary mask.
+
+        Parameters
+        ----------
+        grain_mask_tensor : npt.NDArray
+            Multi class grain mask tensor tensor of shape (N, N, C).
+
+        Returns
+        -------
+        npt.NDArray
+            Combined binary mask of all but the background class (:, :, 0).
+        """
+        assert len(grain_mask_tensor.shape) == 3, f"Tensor not 3D: {grain_mask_tensor.shape}"
+        # Convert to binary in case there are multiple hits in the same pixel. We dont want to have 2s, 3s etc
```

Comment:
Feels like this could either be removed or moved to the doc-string to provide users with more information on the API documentation side of things.